Integrations/integration-AnomaliThreatStream.yml

```diff
@@ -243,6 +243,20 @@ script:
             entry.EntryContext[contextKey] = contexts;
         }
 
+        var dbotScore = 1;
+        if (output['Threat Score'] >= args.threshold) {
+            dbotScore = 3;
+        } else if (output['Threat Score'] > 0) {
+            dbotScore = 2;
+        }
+
+        entry.EntryContext.DBotScore = {
+                                        Indicator: args.value,
+                                        Type: args.type,
```

Comment:
args.type ?
Those should be consts ...

Reply:
This is not really an argument. If you would look at the switch-case, you would find out that each 'type' of command is set into 'args'.